course_info/views.py

```diff
@@ -180,8 +159,8 @@ def editor(request):
         # "lis" appears to be a deliberate misspelling
         course_instance_id = request.POST.get('lis_course_offering_sourcedid')
 
-    # The values we will want to display
-    keys = [
+    # The values we will want to display, in the order we wish them to appear
+    field_names = [
```

Comment:
If we wanted to add a new field to the tool, it looks like there are two places in the code that would need to be changed.  The ```field_names`` list in the editor view function and the ```_FIELD_LABEL_MAP``` at the top of the module. Would it make sense to pull out ```field_names``` and make it a constant at the top of the module? That way if someone needs to add/remove a field or change the order, they only need to look in one place.